jenkinsci/blueocean-plugin
# Task/jenkins 36229

# Description

See https://issues.jenkins-ci.org/browse/JENKINS-36229.

# Submitter checklist
- [x] Apppropriate unit or acceptance tests or explaination to why this change has no tests

To test user permission for a pipeline:

* See https://github.com/jenkinsci/blueocean-plugin/tree/task/JENKINS-36229/blueocean-rest
* On local jenkins, enabled security with no anonymous read access
* GET http://localhost:8080/organizations/jenkins/pipelines/p1/ should give permissions of logged in users all true for read, create, start and stop
* go back to global security, allow anonymous read access and call above pipeline api, log out and as anonymous user call above API, the permission object in pipeline details will be true only for read and false for everything else.


Triggered https://ci.blueocean.io/job/Acceptance%20Tests%20Param/11/

# Reviewer checklist
- [x] Verified that the appropriate tests have been written or valid explaination given

@jenkinsci/code-reviewers @reviewbybees
